coderd/database/migrations/000039_add_workspace_agent_version.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE ONLY workspace_agents ADD COLUMN version text DEFAULT ''::text NOT NULL;
+COMMENT ON COLUMN workspace_agents.version IS 'version tracks the version of the currently running workspace agent. Workspace agents register their version upon start.';
```

Comment:
❤️

Suggestion (non-blocking): Should we title case it? I.e. `Version tracks..`, this way the generated Go comment would look better.